test/test_sparse_csr.py

```diff
@@ -680,6 +680,7 @@ def test_shape(m, n, p, nnz, broadcast, index_dtype, alpha_beta=None):
                   *((torch.complex128,) if CUSPARSE_SPMM_COMPLEX128_SUPPORTED else ()),
                   *torch.testing.get_all_fp_dtypes(include_bfloat16=SM80OrLater,
                                                    include_half=SM53OrLater))
+    @dtypes(())  # since dtypes is mandatory
```

Comment:
Surprised to know that you can pass `()` to `dtypes`

Reply:
It fails for XLA with 
```
Nov 11 17:41:16   File "/var/lib/jenkins/workspace/xla/test/pytorch_test_base.py", line 529, in instantiate_test
Nov 11 17:41:16     if len(dtype_combination) > 1 else dtype_combination[0])
Nov 11 17:41:16 IndexError: tuple index out of range
```

At the following line : https://github.com/pytorch/xla/blob/2014c477ef593561a8bdb45d8e94cde93456a428/test/pytorch_test_base.py#L529

Maybe just use `@dtypesIfCUDA` for `@dtypes` on tests marked with `onlyCUDA`.

Reply:
Using @dtypes() works.